external-crates/move/crates/move-compiler/src/sui_mode/linters/mod.rs

```diff
@@ -83,6 +85,7 @@ pub enum LinterDiagCategory {
     FreezeWrapped,
     CollectionEquality,
     PublicRandom,
+    ConstantNaming,
```

Comment:
We should have a separate category/code system for the linters outside of the sui_mode linters